blackbox/docs/sql/queries.txt

```diff
@@ -220,7 +220,8 @@ Predicates
 Predicates are expressions that evaluate to boolean.
 
 While scalar functions that evaluate to boolean can be used in place of a predicate,
-a predicate cannot be used everywhere, e.g. in the result columns of a select statement.
+not all predicates can be used everywhere, e.g. :ref:`predicates_match` cannot be used in the result columns 
+of a select statement, while :ref:`predicates_is_null` and :ref:`predicates_not_null` can.
```

Comment:
hm, i think it would be better now to document the special case of "match" somewhere else. since the select list constraint has actually nothing to do with predicates anymore